### PR TITLE
fix activity design

### PR DESCRIPTION
**Pull request by szaimen**
| Before | After |
|---|---|
| image | image |

Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: css/style.scss [hunk 1]
```diff
@@ -10,6 +10,7 @@
 	margin-bottom: 5px;
 	margin-left: 5px;
 	text-overflow: ellipsis;
+	word-break: break-word;
```
Comment (Pytal) [inline css/style.scss:13]:
```suggestion
	word-break: normal;
	overflow-wrap: anywhere;
```
Seems `word-break: break-word;` is deprecated https://developer.mozilla.org/en-US/docs/Web/CSS/word-break#values

**Review by Pytal — CHANGES_REQUESTED** [review]

File: css/style.scss (hunk 1 above)
Comment (szaimen) [reply css/style.scss]: done

**Review by Pytal — APPROVED** [review]

**Comment by szaimen** [comment]
The fixup bot is still broken unfortunatley. Does anyone has force-merge privileges?
Maybe @nickvergessen?

**Comment by szaimen** [comment]
@skjnldsv do you have force-merge rights here?

**Comment by szaimen** [comment]
Thanks @nickvergessen ! :)

**Pull request merged**